server/container/guice/cassandra-ldap-guice/src/test/java/org/apache/james/CassandraLdapJamesServerTest.java

```diff
@@ -43,7 +43,7 @@
     private static final String PASSWORD = "secret";
     private static final String DOMAIN = "james.org";
     private static final String ADMIN_PASSWORD = "mysecretpassword";
-    private static Duration slowPacedPollInterval = FIVE_HUNDRED_MILLISECONDS;
+    private static Duration slowPacedPollInterval = ONE_HUNDRED_MILLISECONDS;
     private static ConditionFactory calmlyAwait = Awaitility.with()
         .pollInterval(slowPacedPollInterval)
```

Comment:
be careful, here you change the pollInterval too.

Reply:
That's wanted. Half a second is too long.

Reply:
oops read too fast